docs/components/tasklist/introduction.md

```diff
@@ -1,10 +1,13 @@
 ---
 id: introduction
 title: Introduction
+description: "Tasklist is a tool to work with user tasks in Zeebe."
 ---
 
-Tasklist is a tool to work with [user tasks](/components/modeler/bpmn/user-tasks/user-tasks.md) in Zeebe. It provides a GraphQL API which you can use to build your own application or you can use the general UI we prepared for you.
+Tasklist is a tool to work with [user tasks](/components/modeler/bpmn/user-tasks/user-tasks.md) in Zeebe. Tasklist provides a GraphQL API, which you can use to build your own application, or you can use the general UI we prepared for you.
 
-You can find more about the API [here](../userguide/api/overview), and about the UI [here](../userguide/user-interface/overview).
+Read more on the [API](../userguide/api/overview) and the [UI](../userguide/user-interface/overview).
 
-Tasklist is also available for production use (with support) in the Camunda Cloud offering. If you'd like try out Tasklist in Camunda Cloud, sign up [here](https://accounts.cloud.camunda.io/signup).
+Tasklist is also available for production use (with support) in the Camunda Cloud offering. To try out Tasklist in Camunda Cloud, sign up [here](https://accounts.cloud.camunda.io/signup).
+
+[//]:# (I also think this doc is worth revisiting/fluffing up a bit. Tasklist is capable of so much more than just "working with user tasks." Perhaps even just adding an image or two would encourage users to keep reading the rest of our documentation here.)
```

Comment:
I like your thoughts here and I think you are spot on about the capabilities not being well highlighted.

Reply:
I'm happy to highlight some more of the capabilities within this doc this week.

@ralfpuchert What are your thoughts here? Are there particular features/aspects you would like to see highlighted in this overview?

Reply:
@vsgoulart , @santanaluiz Do you have any contributions to the question above?